CHANGELOG.md

```diff
@@ -0,0 +1 @@
+Remove no-longer-functional Crashlytics handlers
```

Comment:
Nit:
```suggestion
- Removes non-functional Crashlytics handlers.
```
Reworded it to sound slightly better to me, but either way is fine. We should start each line in this with a`-`, however, so that they show up as bullet points on our release notes.